zeus/device/cpu/rapl.py

```diff
@@ -272,10 +280,10 @@ def cpus(self) -> Sequence[cpu_common.CPU]:
     def _init_cpus(self) -> None:
         """Initialize all Intel CPUs."""
         self._cpus = []
-        for dir in sorted(glob(f"{RAPL_DIR}/intel-rapl:*")):
+        for dir in sorted(glob(f"{self.rapl_dir}/intel-rapl:*")):
             parts = dir.split(":")
             if len(parts) > 1 and parts[1].isdigit():
```

Comment:
Okay, I know it's extremely unlikely, but this will break in a system with 10 or more sockets (HPC nodes, basically). Please improve parsing here.

Reply:
Otherwise PR is good to go